src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -381,6 +381,9 @@
   <data name="CmdFocusDesc" xml:space="preserve">
     <value>Launch the window in focus mode</value>
   </data>
+  <data name="CmdSavedLayoutArgDesc" xml:space="preserve">
+    <value>Specify a saved window layout index (from state.json "persistedWindowLayouts") to initially load layout from. </value>

```

Comment:
I wish we could just not have this in the help text at all, because it really is an implementation detail. Maybe we should just make it's description "This parameter is an internal implementation detail and should not be used."

I just don't want folks thinking that they can take a dependency on this (because lawd knows someone will)

Reply:
it _is_ kinda cool to just make a clone of an existing window if you know what you are doing... someone external would absolutely depend on that.

Reply:
Changed to that message.